app/src/main/java/example/chatea/servicecamera/MainActivity.java

```diff
@@ -15,12 +18,29 @@
     private boolean mRecording;
 
     private Button bt_recordingButton;
+    private IntentFilter mIntentFilter;
+    private PhonecallReceiver phonecallReceiver;
+
+    @Override
+    protected void onResume() {
```

Comment:
Registering the BroadcastReceiver for what? MainActivity didn't handle any event from PhonecallReceiver.